backend/README.md

```diff
@@ -1,8 +1,49 @@
 # Backend For FreshNest
-> Node.js
+Prerequisites: 
+- Node.js [v18.4.0](https://nodejs.org/dist/v18.4.0/)
+- [MongoDB](https://www.mongodb.com/try/download/community)
+- npm (included with Node.js)
 
+To run a local development server, follow these steps:
 
-To run the application, simply run the following command:
- ```bash
-node index.js
- ```
+1. Fork the project repository by clicking the "Fork" button at the top-right of this page.<br>
+2. Clone the forked repository to your local machine by running the following command in your terminal:<br>
+
+```
+git clone https://github.com/[your-username]/freshnest.git
+```
+
+Replace [your-username] with your GitHub username.<br>
+
+3. Navigate to the project directory by running the following command in your terminal: <br>
+
+```
+cd /backend/
+```
+
+4. Install the project dependencies by running the following command:
+
+```
+npm install
+```
+
+5. Add an `.env` file in your `/backend` folder and add the below line:
+
+   ```env
+   DATABASE_URL="mongodb://127.0.0.1:27017/fresh-nest" # your MongoDB address with port and collection name
+   ```
+   
+6. Start the development server by running the following command:
+
+```
+npm start
```

Comment:
For local development, I would suggest to run `npm run dev` as we have `nodemon` in our `dev-dependencies`.